### PR TITLE
feat: Documentation spelling policy

### PR DESCRIPTION
**Pull request by dipietroR**
Adding a policy to .perlcriticrc that checks documentation spelling

### CODE REVIEW

File: .perlcriticrc [hunk 1]
```diff
@@ -60,3 +60,9 @@ max_nests = 12
...
...location ip eg po gettext Crowdin hashref https http submodules Config submodule lc lang barcodes packagings HARDCODED hardcoded hashrefs scrypt Scrypt md IPs csv ssconvert openfoodfacts couvercle métal FSC Française VPF viande de porc française Porc Français Origine français Pâtes aromatisées fraise canneberge l'acérola Ingrédients Pulpe poire sucre sirop blé céléri GDSN Equadis CodeOnline nodejs gtin csv CIN CIC NGO EREN PNNS et Santé matche undef NaN kcal AGS saturés barquette de CIQUAL AgriBalyse LCA EPI nutriscore msgctxt userid UUID appid uuid FZF RTFSG openfoodfacts subfield Nutriscore Santé publique kJ colza Valeur EXIF PNG incrontab gif jpeg jpf png heic Tesseract Robotoff jpg tesseract sprintf nutri Nutri www formatter ur Storable uncharacterized Agribalyse malus Canonicalizes canonicalized Programme Envol GSDN Carrefour multiline Multiline backticks scanbot taxonomize taxonomized gnumeric sto tagid canonicalize str Lowercased stopwords greated rescale overriden contenant maltodextrine vitamines stabilisant glycérol gélifiant pectine acidifiant acide citrique arôme naturel poudre d'acérola acérola Fabriqué dans un utilisant Allergènes flavouring flavourings dont Catalogue
```
Comment (Ban3) [inline .perlcriticrc:67]: Maybe `stop_words_file` for such a long list of words?

Comment (dipietroR) [reply .perlcriticrc]: How's this?

Comment (Ban3) [reply .perlcriticrc]: LGTM

File: stop_words.txt [hunk 2]
```diff
@@ -0,0 +1,184 @@
+JSON
```
Comment (alexgarel) [inline stop_words.txt:1]: if it's not too much to ask, I would keep this file in alphabetical order !

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Pull request merged**